src/sentry/static/sentry/app/components/bulkNotice.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Alert from 'app/components/alert';
+import {tn, tct, t} from 'app/locale';
+import {defined} from 'app/utils';
+
+export function getEverythingSelectedText(allRowsCount?: number, bulkLimit?: number) {
```

Comment:
shouldn't it be better to create another file for these functions, like ` utils.tsx`? or maybe just move the exports to the end of this file?